src/app/Home/operations/home.data.js

```diff
@@ -1 +1,64 @@
+// /* eslint-disable max-len */
+// import { put, takeEvery, call } from 'redux-saga/effects';
 
+// import actions from './actions';
+// import { CATEGORIES, LOG_POINTS } from './constants';
```

Comment:
What is with this file?

Reply:
@kimani-ndegwa I have deleted this file and everything on the Home folder. This is because we will no longer need them.